source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/table/column-data.ts

```diff
@@ -16,7 +16,8 @@ export const possibleColumnsByType = {
     [COLUMN_KEY.PROJECT_OWNER]: {
       id: COLUMN_KEY.PROJECT_OWNER,
       i18nKey: 'projectTable.columns.owner',
-      propertyPath: 'owner.name'
+      propertyPath: 'owner.name',
+      // sortable: true
```

Comment:
this is pending a custom sort implementation, but this field should probably be sortable in the future